force-app/src/classes/Mock.cls

```diff
@@ -9,9 +9,6 @@ public class Mock implements System.StubProvider {
   public Object stub { get; private set; }
   private Map<String, MethodSpy> spies = new Map<String, MethodSpy>();
 
-  private Mock() {
-  }
-
```

Comment:
**comments:** I want to bring context behind this change

The idea behind it was to show it is not possible to instanciate the mock outside the static `forType` method.
It is not necessary, and was here just for clarity

Reply:
I get that, I really do. I think if there wasn't _another_ private constructor in this class, its presence would totally make sense.